bridges/snowbridge/pallets/outbound-queue-v2/src/lib.rs

```diff
@@ -315,7 +321,8 @@ pub mod pallet {
 			Self::deposit_event(Event::MessagesCommitted { root, count });
 		}
 
-		/// Process a message delivered by the MessageQueue pallet
+		/// Process a message delivered by the MessageQueue pallet.
+		/// IMPORTANT!! This method does not roll back storage changes on error.
```

Comment:
So it should be called in a transactional context from outside? Or we don't want it to rollback at all?

Reply:
I think we don't want to roll back at all. Maybe @yrong can confirm?

Reply:
no rollback required - all validations happen before any storage changes, if all goes well, storage gets updated, otherwise the message is dropped with no storage side-effects